### ISSUE TITLE
Set database date data timezone 

### ISSUE THREAD

**Issue by razafinr** [logstash-plugins/logstash-input-jdbc#89]
I am using the logstash jdbc-input plugin to import data from my database to Elasticsearch. 

In the old jdbc river, I think that it was possible to set the timezone used by the database for date data. Is it possible to set it with the logstash jdbc plugin ? 

For the moment the dates in my database are in UTC but the jdbc-plugin use the local timezone of my machine when exporting them and tries to convert it again in UTC so all my dates are wrong.

**Comment by razafinr** [comment]
The workaround of my issue is to convert all dates to VARCHAR when getting them from database.

Then it is possible to apply a date filter on EACH of them. I can not apply directly the date filter without converting the dates to VARCHAR because the dates are retrieved as Ruby date object by the jdbc input so they can not be filtered directly. 

This seems really painful as solution, as we can just have a parameter for the jdbc input to specify the timezone of the database dates.

**Comment by suyograo** [comment]
@razafinr https://github.com/logstash-plugins/logstash-input-jdbc/pull/83 added support to convert to Logstash::Timestamp from Ruby timestamp. we should fix the UTC aspect

**Comment by untergeek** [comment]
Closed by #95

**Comment by razafinr** [comment]
Thank you for the fix.

I think there is one dependency missing (I am totally new to Ruby): `tzinfo-data`.

When I tried to run the master branch after installing it, I got this issue: `TZInfo::DataSourceNotFound`

The solution was then referred here: https://github.com/tzinfo/tzinfo/wiki/Resolving-TZInfo::DataSourceNotFound-Errors

I added `tzinfo-data` to the gemspec and reinstalled the plugin and it worked like charm.
